dashboard/modules/job/common.py

```diff
@@ -25,6 +25,7 @@
 # In order to get information about SupervisorActors launched by different jobs,
 # they must be set to the same namespace.
 SUPERVISOR_ACTOR_RAY_NAMESPACE = "SUPERVISOR_ACTOR_RAY_NAMESPACE"
+JOB_LOGS_PATH_TEMPLATE = "job-driver-{job_id}.log"
```

Comment:
Use  submission id instead since it is technically a submission id? 
```suggestion
JOB_LOGS_PATH_TEMPLATE = "job-driver-{submission_id}.log"
```